server/game/cards/03_TWI/events/DisruptiveBurst.ts

```diff
@@ -0,0 +1,22 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { EventCard } from '../../../core/card/EventCard';
+
+export default class DisruptiveBurst extends EventCard {
+    protected override getImplementationId () {
+        return {
+            id: '3596811933',
+            internalName: 'disruptive-burst',
+        };
+    }
+
+    public override setupCardAbilities () {
+        this.setEventAbility({ title: 'Give each enemy unit -1/-1 for this phase',
```

Comment:
nit: the spacing for properties and open / close braces and parens in this file is inconsistent with all the others